src/test/java/database/helper/ValidatorTest.java

```diff
@@ -0,0 +1,98 @@
+package database.helper;
+
+import database.service.JsonDatabaseService;
+import database.entity.Student;
+import database.exception.EmptyValueException;
+import database.exception.IncorrectPropertyNameException;
+import database.exception.IncorrectValueTypeException;
+import database.exception.NullPropertyNameOrValueException;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ValidatorTest {
```

Comment:
`ValidatorTest` means that you test `Validator` class. I see that here you test `JsonDatabaseService` class. Please move these methods back to the `JsonDatabaseServiceTest` class and write appropriate tests for `Validator`.